docs/ECS Fargate Tutorials/flyway.md

```diff
@@ -0,0 +1,266 @@
+  ```
+  You will be able to see your PostgreSQL connection information when you press Connect.
+
+  </TabItem>
+</Tabs>
+
+<img width={440} src={require('@site/static/screenshots/credentials.png').default} />
+
+Make sure to copy the PostgreSQL connection string as you will not see it again.
+
+## Add the necessary cloud services to the hosted database
+
+1. Many different clients can be used to [connect](/connect) to a PostgreSQL database. For this tutorial, we'll use the standard `psql` CLI client. If you need to install `psql`, follow the instructions for your corresponding OS [here](https://www.postgresql.org/download/).
+
+2. The first migration calls the `iasql_install` SQL function to install [modules](/module) into the hosted database.
+
+```sql title="ecs-fargate-examples/flyway/infra/V1__install.sql"
+SELECT * from iasql_install(
+   'aws_cloudwatch@0.0.1',
+   'aws_ecr@0.0.1',
+   'aws_ecs_fargate@0.0.1',
+   'aws_elb@0.0.1',
+   'aws_security_group@0.0.1',
+   'aws_vpc@0.0.1'
+);
+```
+
+If the function call is successful, it will return a virtual table with a record for each new table in your database under `table_name` and the number of existing resources or records imported from the account under `record_count`.
+
+```sql
+       module_name        |          table_name           | record_count 
+--------------------------+-------------------------------+--------------
+ aws_cloudwatch@0.0.1     | log_group                     |            0
+ aws_ecr@0.0.1            | public_repository             |            0
+ aws_ecr@0.0.1            | repository                    |            1
+ aws_ecr@0.0.1            | repository_policy             |            0
+ aws_security_group@0.0.1 | security_group                |            2
+ aws_security_group@0.0.1 | security_group_rule           |            0
+ aws_vpc@0.0.1            | vpc                           |            1
+ aws_vpc@0.0.1            | subnet                        |            3
+ aws_elb@0.0.1            | load_balancer                 |            0
+ aws_elb@0.0.1            | target_group                  |            0
+ aws_elb@0.0.1            | listener                      |            0
+ aws_elb@0.0.1            | load_balancer_security_groups |            0
+ aws_ecs_fargate@0.0.1    | cluster                       |            0
+ aws_ecs_fargate@0.0.1    | service                       |            0
+ aws_ecs_fargate@0.0.1    | task_definition               |            0
+ aws_ecs_fargate@0.0.1    | container_definition          |            0
+ aws_ecs_fargate@0.0.1    | service_security_groups       |            0
+(17 rows)
+```
+
+## Connect to the hosted db and provision cloud resources in your AWS account
+
+1. Get a local copy of the [ECS Fargate examples repository](https://github.com/iasql/ecs-fargate-examples)
+
+```bash
+git clone git@github.com:iasql/ecs-fargate-examples.git
+cd ecs-fargate-examples/flyway
```

Comment:
Since they're only interested in the `flyway` example, maybe instead we get them to filter out the stuff they don't care about?

```suggestion
git clone git@github.com:iasql/ecs-fargate-examples.git my_project
cd my_project
git filter-branch --subdirectory-filter flyway
```

Reply:
hmm good point